charts/opentelemetry-kube-stack/templates/clusterrole.yaml

```diff
@@ -17,6 +17,7 @@ rules:
   - pods
   - events
   - secrets
+  - persistentvolumeclaims
```

Comment:
@jaronoff97 is this another case where the operator needs the same permissions as the collector's receivers?

Reply:
I believe so, yes

Reply:
@jaronoff97 @TylerHelmuth Can this be resolved?